internal/wasm/sdk/opa/opa.go

```diff
@@ -45,7 +45,7 @@ type Result struct {
 // initialized before invoking the Eval.
 func New() *OPA {
 	opa := &OPA{
-		memoryMinPages: 16,
+		memoryMinPages: 129,
```

Comment:
I'm not sure I understand why this has to change. Can you include a comment here why it needs to be increased by almost 10x? Is there some significance to 129 (e.g., instead of a power of two)

Reply:
Yep I don't want to have to change that either 😸 this is a remnant from debugging, sorry.